src/CalcManager/sal_cross_platform.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+// Empty macro definitions for source annotations
+
+#define _In_opt_
+#define _Out_opt_
+#define _In_
+#define _Out_
+#define _Inout_
+#define __in_opt
```

Comment:
There's another one being added in #304: `_Frees_ptr_opt_`

Reply:
Added, thanks.